docs/module-hooks.md

```diff
@@ -175,3 +175,20 @@ Extra [devspace profiles](https://www.devspace.sh/docs/5.x/configuration/profile
 
 {{ stencil.AddToModuleHook "github.com/getoutreach/stencil-golang" "devspace.profiles" (stencil.ApplyTemplate "ingestDevspaceProfile" | fromYaml) }}
 ```
+
+### `Makefile.commands`
+
+**Type**: `string`
+
+**File**: `Makefile`
+
+Extra commands to add to the root Makefile
+
+```tpl
```

Comment:
This example should include an actual sample of how you would apply this. This should include the call to `stencil.AddToModuleHook`. 

Ideally, this example should come from your test case of running this inside a stencil module. You can always use `replacement` inside `service.yaml` to test this locally.

Reply:
...I failed to capture the surrounding context :(. Thanks.  Let me try that again.

Reply:
Better?

Reply:
Perfect! Thanks for expanding this.